questionpy_sdk/webserver/templates/attempt.html.jinja2

```diff
@@ -7,40 +7,53 @@
 {% endblock %}
 
 {% block content %}
-    <div class="container-question-preview">
-        <form id="question-preview" action="/submit" data-success-action="reload">
-            {{ question_html|safe }}
-        </form>
-        <button id="save-attempt-button" type="submit" data-route="/attempt/save" form="question-preview" {% if form_disabled %}disabled{% endif %}>Save</button>
-        <button id="submit-attempt-button" type="submit" data-route="/attempt" form="question-preview" {% if form_disabled %}disabled{% endif %}>Save and submit</button>
-        <button id="restart-attempt-button" type="button" data-route="/attempt/restart" {% if not form_disabled %}disabled{% endif %}>Restart</button>
-        <button id="edit-attempt-button" type="button" data-route="/attempt/edit" {% if not form_disabled %}disabled{% endif %}>Edit</button>
+    <div class="container-question-info">
+        <p class="score">
+            {{ ("Score: " ~ attempt.score) if attempt.score is defined else "Not yet scored" }}
```

Comment:
Sowohl `is defined` als auch `is not none` sind leider ein semantischer Unterschied.

`not in (none, undefined)` scheint es zu tun

Reply:
Dann steht halt "Not yet scored" da, wenn ein AttemptScoredModel mit `score=None` zurück kommt.